dla_cnn/desi/DesiMock.py

```diff
@@ -64,14 +65,25 @@ def read_fits_file(self, spec_path, truth_path, zbest_path):
         ivar_r = spec[9].data.copy()
         ivar_z = spec[14].data.copy()
         error = 1./np.sqrt(np.hstack((ivar_b,ivar_r,ivar_z)))
-        self.split_point_br = len(flux_b)
-        self.split_point_rz = len(flux_b)+len(flux_z)
+        self.split_point_br = flux_b.shape[1]
+        self.split_point_rz = flux_b.shape[1]+flux_r.shape[1]
         z_qso = zbest[1].data['Z'].copy()
         ra = spec[1].data['TARGET_RA'].copy()
         dec = spec[1].data['TARGET_DEC'].copy()
 
         self.data = {spec_id[i]:{'FLUX':flux[i],'ERROR': error[i], 'z_qso':z_qso[i] , 'RA': ra[i], 'DEC':dec[i], 'DLAS':spec_dlas[spec_id[i]]} for i in range(len(spec_id))}
 
+        if normalize:
+            err_spec = ''
+            for spec,data in self.data.items():
+                rest_wavelength = self.wavelength/(data['z_qso']+1)
+                if rest_wavelength[0]<1450:
+                    test = (rest_wavelength>=1400)&(rest_wavelength<=1500)
+                    data['FLUX'] = data['FLUX']/np.abs((np.median(data['FLUX'][test])))
```

Comment:
Why do you take `np.abs` of the median?  Surely it should never be negative.

Reply:
How about of instead of actually applying the normalization that you record it in `self.data` and we can choose to use it later when generating the `Sightline` object?

This would actually allow you to calculate both the normalization values we are considering.

Reply:
Also, we need to renormalize the error array by the same factor to maintain the signal/noise.